tests/trace_service.py

```diff
@@ -0,0 +1,88 @@
+import concurrent
+import grpc
+from opentelemetry.proto.collector.trace.v1 import trace_service_pb2
+from opentelemetry.proto.collector.trace.v1 import trace_service_pb2_grpc
+import pytest
+import subprocess
+import time
+
+
+class TraceService(trace_service_pb2_grpc.TraceServiceServicer):
+    spans = []
+
+    def Export(self, request, context):
+        self.spans.append(request.resource_spans)
+        return trace_service_pb2.ExportTracePartialSuccess()
+
+    def get_span(self, timeout=0.5, clear=True):
```

Comment:
I don't think we need to customize `timeout` or `clear` here.

Also, this method should assume almost immediate span delivery, so time can be reduced, say, to 10 times for 1ms.